tutorials/integrations/main.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import importlib
+import logging
+from integrations.common.helpers import write_results, read_models_list, parse_results
+from integrations.common.helpers import find_modules
+from integrations.common.consts import MODEL_NAME, MODEL_LIBRARY, OUTPUT_RESULTS_FILE, TARGET_PLATFORM_NAME, \
+    TARGET_PLATFORM_VERSION
+
+
+def argument_handler():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--model_name', '-m', type=str, required=False,
+                        help='The name of the pre-trained model to run')
+    parser.add_argument('--model_library', type=str, required=False,
+                        help='The library that contains the pre-trained model')
+    parser.add_argument('--validation_dataset_folder', type=str, required=False,
+                        help='Path to the validation dataset')
+    parser.add_argument('--representative_dataset_folder', type=str, required=False,
+                        help='Path to the representative dataset used for quantization')
+    parser.add_argument('--num_representative_images', type=int, default=1024,
+                        help='Number of images for representative dataset')
+    parser.add_argument('--dataset_name', type=str, default='',
+                        help='Datset name for comment')
```

Comment:
what d you mean "for comment"? Is it not used to choose the dataset? (if not then it's a bit confusing)